components/eamxx/src/share/property_checks/field_nan_check.cpp

```diff
@@ -47,7 +47,7 @@ PropertyCheck::ResultAndMsg FieldNaNCheck::check_impl() const {
   switch (layout.rank()) {
```

Comment:
Can you add a comment here too, saying that "we can't be sure that the field has contiguous allocation, so we use get_strided_view"?